src/components/SendSykmelding/FormSections/Arbeidssituasjon/Egenmelding/EgenmeldingDatesPickerSubField.module.css

```diff
@@ -1,15 +1,3 @@
-.velgDager {
-    margin: var(--a-spacing-4) 0;
-}
-
-.twoDatepickers {
-    display: flex;
-}
-
-.datoError {
-    margin-bottom: var(--a-spacing-4);
-}
-
 @media (max-width: 350px) {
     .velgDager :global(.navds-date__standalone-wrapper) {
```

Comment:
Korleis blir dette med Tailwind? For no blir ikkje `.velgDager` brukt i komponenten.